### ISSUE TITLE
Flags input results in unrecognized argument error

### ISSUE THREAD

**Issue by DiannaBanana** [google-github-actions/create-cloud-deploy-release#27]
### TL;DR

When creating a new Cloud Deploy release with this GH action and setting some `flags` input, the action fails and results in an unrecognized argument error instead of using the specified flag.

### Expected behavior

A successful Cloud Deploy release was created using the `skaffold_preview` version that was specified through the `--skaffold-version` flag.

### Observed behavior

The GH action fails to create a Cloud Deploy release because it results in an unrecognized argument error. Flags input is getting parsed strangely.

### Action YAML

```yaml
name: Create a release in Cloud Deploy
on:
  push:
    branches:
      - main
env:
  PIPELINE: deploy-test
  REGION: us-east4

jobs:
  deploy-test:
    permissions:
      contents: 'read'
      id-token: 'write'

    runs-on: ubuntu-latest
    steps:
    - name: Checkout repo
      uses: actions/checkout@v3.3.0

    - name: 'Google auth'
      id: 'auth'
      uses: 'google-github-actions/auth@v1'
      with:
        workload_identity_provider: '${{ secrets.WIF_PROVIDER }}'
        service_account: '${{ secrets.WIF_SERVICE_ACCOUNT }}'

    - name: 'Create release name'
      run: |-
        echo "RELEASE_NAME=release-${GITHUB_SHA::7}" >> ${GITHUB_ENV}

    - name: 'Create Cloud Deploy release'
      id: 'release'
      uses: 'google-github-actions/create-cloud-deploy-release@v0.1.1'
      with:
        delivery_pipeline: ${{ env.PIPELINE }}
        name: ${{ env.RELEASE_NAME }}
        region: ${{ env.REGION }}
        description: Test-Deployment
        skaffold_file: 'config/skaffold.yaml'
        flags: '--skaffold-version=skaffold_preview'

    - name: 'Report Cloud Deploy release'
      run: |-
        echo "Created release ${{ steps.release.outputs.name }} "
        echo "Release link ${{ steps.release.outputs.link }} "
```

### Log output

```text
Run google-github-actions/create-cloud-deploy-release@v0.1.1

/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/10d2e040 -f /home/runner/work/_temp/0dbbe185
Successfully authenticated

Running: gcloud deploy releases create Release-18ff0ee --delivery-pipeline deploy-test --region us-east4 --skaffold-file config/skaffold.yaml --annotations commit=https://github.com/org/repo/commit/18ff0ee18ff0ee18ff0ee18ff0ee18ff0ee18ff0ee,git-sha= ecdeca633a230bfade4cc8195ae23af030922319 --labels managed-by=github-actions --flags --skaffold-version skaffold_preview --format json

Error: create-cloud-deploy-release failed with: failed to execute gcloud command `gcloud deploy releases create Release-18ff0ee --delivery-pipeline deploy-test --region us-east4 --skaffold-file config/skaffold.yaml --annotations commit=https://github.com/org/repo/commit/18ff0ee18ff0ee18ff0ee18ff0ee18ff0ee18ff0ee,git-sha= ecdeca633a230bfade4cc8195ae23af030922319 --labels managed-by=github-actions --flags --skaffold-version skaffold_preview --format json`: ERROR: (gcloud.deploy.releases.create) unrecognized arguments:
  --flags (did you mean '--flags-file'?)

  To search the help text of gcloud commands, run:
  gcloud help -- SEARCH_TERMS
```

### Additional information

This might be happening here: https://github.com/google-github-actions/create-cloud-deploy-release/blob/main/src/main.ts#L156-L157 
where the `--flags` string is getting pushed into the gcloud command before appending the actual flags that have been passed in. It also seems like the flags are being parsed weirdly, since the input was originally `--skaffold-version=skaffold_preview` but the final command that was executed contains `--skaffold-version skaffold_preview`, which changed the `=` to a space.

**Comment by henrybell** [comment]
Thanks @DiannaBanana! It was a parsing issue as you mentioned, along with a test that had matching (incorrect) validation. I have a fix that I am testing, which should be in shortly.

**Comment by DiannaBanana** [comment]
Awesome, thanks @henrybell!

**Comment by henrybell** [comment]
Patch release upcoming shortly -- thanks again for the helpful detail in the issue! Much appreciated.
